### PR TITLE
Mark selected notifications as unread in admin

### PR DESCRIPTION
**Pull request by s0th1nk**
This is helpful when you want to resend notification.

### CODE REVIEW

**Comment by pandafy** [comment]
Can you please add tests for this feature?

File: notifications/admin.py [hunk 1]
```diff
@@ -6,14 +6,18 @@
 
 Notification = load_model('notifications', 'Notification')
 
+def mark_unread(modeladmin, request, queryset):
+    queryset.update(unread=True)
+mark_unread.short_description = "Mark selected notifications as unread"
```
Comment (AlvaroLQueiroz) [inline notifications/admin.py:11]:
```suggestion
mark_unread.short_description = gettext_lazy('Mark selected notifications as unread')
```

File: notifications/admin.py [hunk 2]
```diff
@@ -6,14 +6,18 @@
```
Comment (AlvaroLQueiroz) [inline notifications/admin.py:6]:
```suggestion
from django.utils.translation import gettext_lazy

```

**Comment by AlvaroLQueiroz** [comment]
@s0th1nk can you rebase your branch, please?

**Pull request closed without merging**